package.json

```diff
@@ -2,13 +2,18 @@
   "private": true,
   "description": "blockprotocol monorepo",
   "type": "module",
-  "workspaces": [
-    "crates/*",
-    "packages/*",
-    "packages/@blockprotocol/*",
-    "packages/@local/*",
-    "site"
-  ],
+  "workspaces": {
+    "packages": [
+      "crates/*",
+      "packages/*",
+      "packages/@blockprotocol/*",
+      "packages/@local/*",
+      "site"
+    ],
+    "nohoist": [
+      "**/@types/prismjs"
```

Comment:
Prismjs typings refer to DOM nodes, so when they are hoisted, workspaces with node-only tsconfig fail with this error:

```
../../node_modules/@types/prismjs/index.d.ts(73,46): error TS2304: Cannot find name 'ParentNode'.
```